.github/workflows/publish-commit-bottles.yml

```
@@ -335,11 +335,27 @@ jobs:
 
           echo "head_sha=$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
 
-      - name: generate build provenance
-        uses: github-early-access/generate-build-provenance@main
+      - name: Generate build provenance
+        uses: actions/attest-build-provenance@v1
         with:
           subject-path: '${{steps.pr-pull.outputs.bottle_path}}/*.tar.gz'
-  
+
+      - name: Upload bottles to GitHub Packages
+        id: pr-upload
+        working-directory: ${{steps.set-up-homebrew.outputs.repository-path}}
+        env:
+          BREWTESTBOT_NAME_EMAIL: "BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>"
+          HOMEBREW_GITHUB_PACKAGES_USER: brewtestbot
+          HOMEBREW_GITHUB_PACKAGES_TOKEN: ${{secrets.HOMEBREW_CORE_GITHUB_PACKAGES_TOKEN}}
+        run: |
+          # Don't quote arguments that might be empty; this causes errors.
```

```suggestion
          set -euo pipefail

          # Don't quote arguments that might be empty; this causes errors.
```

may be necessary to make this fail on failure

---

It's actually not a shell thing, it's about the workflow interpolation (`${{ }}`) so that can't be fixed by tweaking the shell.